src/api/app/assets/javascripts/webui/responsive_ux.js

```diff
@@ -47,3 +47,15 @@ function toggleTabs(tabLinkContainerId) { // jshint ignore:line
     currentTabLink.addClass('active');
   });
 }
+
+function setFormValidation(element, messages) { // jshint ignore:line
+  element.addClass('is-invalid');
+  element.after("<div class='invalid-feedback'>"+ messages + "</div>");
+}
+
+function resetFormValidation() { // jshint ignore:line
+  $('.in-place-edition form .is-invalid').each(function(){
+    $(this).toggleClass('is-invalid');
+    $(this).siblings('.invalid-feedback').remove();
+  });
+}
```

Comment:
Hm, isn't this something that happens automatically with UJS?

Reply:
The validations weren't shown, so I had to handle it manually. But let me research a bit more...